src/Cassandra.IntegrationTests/TestClusterManagement/SimulacronManager.cs

```diff
@@ -30,7 +30,7 @@ public static Uri BaseAddress
 
         public void Start()
         {
-            if (_started)
+            if (IsUp())
```

Comment:
I think we should use a different flag here, like `_init` or whatever, checking that it's started just once.
If the process has exited, it will fail on the next `POST` or `GET` request, which is the desired behaviour.